drivers/hmis/app/graphql/types/hmis_schema/ce_referral_step.rb

```diff
@@ -21,6 +21,7 @@ class HmisSchema::CeReferralStep < Types::BaseObject
     field :assignees, [Application::User], null: false, description: 'User(s) currently assigned to this step'
     field :updated_by, Application::User, null: true
     field :updated_at, GraphQL::Types::ISO8601DateTime, null: true
+    field :can_current_user_perform, Boolean, null: false
```

Comment:
Didn't occur to me until reviewing the frontend, but this does break our API pattern for permissions. More in line would be to resolve an access object:
```
referralStep {
   access {
       canPerformStep
   }
}
```
I think that would be relatively simple to do:
```ruby
access_field do
      field :can_perform_step, Boolean, null: false
end

def can_perform_step # no need to call it "current user", the access fragment implies that
    true
end
```